.eslintrc.js

```diff
@@ -175,10 +175,10 @@ module.exports = {
                   '!src/core/server/*.test.mocks.ts',
 
                   'src/plugins/**/public/**/*',
-                  '!src/plugins/**/public/index*',
+                  '!src/plugins/**/public/index.{ts,tsx}',
```

Comment:
Do you think we should include `js` in here and below?

Reply:
I guess not. In `src/plugins` folder, it is highly recommended to use TypeScript files. In most cases, index files aren't js files while all other files are ts files.

Reply:
On one hand that's the recommendation.
However, there is one folder in the `plugins` directory that has js files (`cron_editor`) and I'm not sure if we should be restricting users this way in general.

Bottom line: I'm good with whatever @spalger decides.

Reply:
Yeah, I'm not opposed to adding a lint rule that enforces index.ts files instead of index.js files, but this isn't the right place to implement that limitation. Please add js to the list

Reply:
Done.